### DISCUSSION TITLE
Closing Applications and Going Forward

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/caMicroscope#955] [General]
Hello all,

Just letting you know that we were actually a little late closing applications, so sorry we didn't give a warning/heads up.

We only are able to mentor one student this term, and this is always a nearly impossible choice. We won't be able to reveal any information about the selections until outreachy does in May.

That said...
A lot of people here have made valuable contributions. I want to thank you on behalf of caMicroscope users, but I also want to ask a question! 

How else can we recognize/give back to you all? That is, pretending that we weren't able to select anyone, what kinds of things can we do to help this process feel more valuable for participating.

Of course, reviewing the backlog of PRs then doing a release so you're in the commit history on the main branch is at the top of this list. But I think there are other things we can do. Some ideas we've considered (given that we don't have much/any extra funding to allocate as far as I know):
* Invite people who consistently write good issues to join the organization at triage access (also help onboarding, many of you picked up a lot of the slack I left for onboarding these past months)
* Write letters of recommendation
* Suggest you to our collaborators if your interests align with one of their projects (probably hardest to do, but we can see)

Please comment/discuss if you have suggestions. It's ok if the answer is nothing more; in that case we are glad that you stopped by to help usability in bio research in the ways that you did! (and if you want more, than this appreciation extends to the future too!)

Thank you again,
- Birm

**Comment by yusuf-khaan** [comment]
@birm We know that many of us tried to contribute to this project as said only one can be allocated but this project has helped me learn so much lot more than i previously knew and has intrigued our interest, i honestly loved working on it, I was currently trying to build model to include api support for high dimesional images and has given me so much knowledge on its implementation, I was able to discover so many libraries, its implementation, Those of us who loved this project will continue contributing to this project

Thanks

**Comment by Akhilesh-max** [comment]
Hi @birm, Greetings!
 Is the case same with GSoC?

**Comment by gbengaoluwadahunsi** [comment]
Hello @birm , 

I had a great time getting feedback from you and also working and learning from other contributors.

That being said, I want to suggest if you have other internship openings in your inner circle, you can kindly recommend contributors from this project to them.

Thank you.
